### PR TITLE
feat(EMS-621 EMS-771): move insurance eligibility exit pages into insurance route

### PR DESCRIPTION
**Pull request by ttbarnes**
This PR moves the 'speak to UKEF EFM' and 'apply offline' exit pages in insurance eligibility, into a generic insurance route so that the URL makes sense in both contexts.

Previously:
- http://exip.com/insurance/eligibility/speak-to-a-UKEF-EFM
- http://exip.com/insurance/eligibility/apply-offline

Now:
- http://exip.com/insurance/speak-to-a-UKEF-EFM
- http://exip.com/insurance/apply-offline

### CODE REVIEW

**Review by Zainzzkk — APPROVED** [review]
LGTM

File: src/ui/server/controllers/insurance/speak-to-ukef-efm/index.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { PAGES } from '../../../content-strings';
+import { TEMPLATES } from '../../../constants';
+import { Request, Response } from '../../../../types';
+import corePageVariables from '../../../helpers/page-variables/core/insurance';
+
+const get = (req: Request, res: Response) => {
+  const EXIT_REASON = req.flash('exitReason');
+
```
Comment (abhi-markan) [inline src/ui/server/controllers/insurance/speak-to-ukef-efm/index.ts:8]: Any possiblity of `req` being empty, do we need to redirect user to a problem page?

**Review by abhi-markan — CHANGES_REQUESTED** [review]
Good PR, one minor comment to address please.

**Comment by ttbarnes** [comment]
Merging this in - concern that @abhi-markan raised is extremely unlikely if not impossible and the whole UI server would probably not be working.

File: src/ui/server/controllers/insurance/speak-to-ukef-efm/index.ts (hunk 1 above)
Comment (ttbarnes) [reply src/ui/server/controllers/insurance/speak-to-ukef-efm/index.ts:8]: Marking as resolved so we can merge - extremely unlikely if not impossible and the whole UI server would probably not be working. Let me know if still concerned @abhi-markan can have a chat or look into what we could do with all controllers.

**Pull request merged**